datasets/babi_qa/README.md

```diff
@@ -0,0 +1,1017 @@
+| shuffled-qa10     |     200 |    200 |          -   |
+| shuffled-qa11     |     200 |    200 |          -   |
+| shuffled-qa12     |     200 |    200 |          -   |
+| shuffled-qa13     |     200 |    200 |          -   |
+| shuffled-qa14     |     200 |    200 |          -   |
+| shuffled-qa15     |     250 |    250 |          -   |
+| shuffled-qa16     |    1000 |   1000 |          -   |
+| shuffled-qa17     |     125 |    125 |          -   |
+| shuffled-qa18     |     198 |    199 |          -   |
+| shuffled-qa19     |    1000 |   1000 |          -   |
+| shuffled-qa20     |      94 |     93 |          -   |
+| shuffled-10k-qa1  |    2000 |    200 |          -   |
+| shuffled-10k-qa2  |    2000 |    200 |          -   |
+| shuffled-10k-qa3  |    2000 |    200 |          -   |
+| shuffled-10k-qa4  |   10000 |   1000 |          -   |
+| shuffled-10k-qa5  |    2000 |    200 |          -   |
+| shuffled-10k-qa6  |    2000 |    200 |          -   |
+| shuffled-10k-qa7  |    2000 |    200 |          -   |
+| shuffled-10k-qa8  |    2000 |    200 |          -   |
+| shuffled-10k-qa9  |    2000 |    200 |          -   |
+| shuffled-10k-qa10 |    2000 |    200 |          -   |
+| shuffled-10k-qa11 |    2000 |    200 |          -   |
+| shuffled-10k-qa12 |    2000 |    200 |          -   |
+| shuffled-10k-qa13 |    2000 |    200 |          -   |
+| shuffled-10k-qa14 |    2000 |    200 |          -   |
+| shuffled-10k-qa15 |    2500 |    250 |          -   |
+| shuffled-10k-qa16 |   10000 |   1000 |          -   |
+| shuffled-10k-qa17 |    1250 |    125 |          -   |
+| shuffled-10k-qa18 |    1978 |    199 |          -   |
+| shuffled-10k-qa19 |   10000 |   1000 |          -   |
+| shuffled-10k-qa20 |     933 |     93 |          -   |
+
+
+## Dataset Creation
+
+### Curation Rationale
+
+[More Information Needed]
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+Code to generate tasks is available on [github](https://github.com/facebook/bAbI-tasks)
+
+#### Who are the source language producers?
+
+[More Information Needed]
+
+### Annotations
+
+#### Annotation process
+
+[More Information Needed]
+
+#### Who are the annotators?
+
+[More Information Needed]
+
+### Personal and Sensitive Information
+
+[More Information Needed]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[More Information Needed]
+
+### Discussion of Biases
+
+[More Information Needed]
+
+### Other Known Limitations
+
+[More Information Needed]
+
+## Additional Information
+
+### Dataset Curators
+
+Jesse Dodge and Andreea Gane and Xiang Zhang and Antoine Bordes and Sumit Chopra and Alexander Miller and Arthur Szlam and Jason Weston, at Facebook Research.
+
+### Licensing Information
+
+The license comes with the compressed file containing the dataset:
+```
+CC License
+
+bAbI tasks data
+
+Copyright (c) 2015-present, Facebook, Inc. All rights reserved.
+
+Creative Commons Legal Code
```

Comment:
As in the other babi datasets, you can just add the license name here instead of adding its full content, since it's already in the script